sample-kotlin/src/main/kotlin/social/bigbone/sample/GetInstanceInfo.kt

```diff
@@ -1,9 +1,17 @@
 package social.bigbone.sample
 
-import com.google.gson.Gson
+import kotlinx.serialization.json.Json
 import social.bigbone.MastodonClient
+import social.bigbone.api.entity.Instance
 
 object GetInstanceInfo {
+
+    private val JSON_SERIALIZER: Json = Json {
```

Comment:
Can we use the same serializer config from `JsonSerializer.kt` eventually?

Reply:
Depends on if we want to share it with consumers of this library. I was more of the opinion we should not and keep it private/an implementation detail.

What do you think? I know that GSON had been an `api`, and thus shared with consumers, dependency.

Reply:
Actually, the serializer should not be exposed to the consumer in any way, and thus be kept internal. I just wasn't sure if there's a way of sharing the serializer with the test classes without exposing it to the library consumer at ghe same time. 

Anway, your PR already is a big improvement, as we no longer expose the serializer to the consumer 👍🏻

We can keep this as it is. Fine for me 😊

Reply:
Not as far as I’m aware as, in the end, the sample apps are mere consumers of the library as well and would thus receive the same shared dependencies as any other consumer would. 🤔